### PR TITLE
Add plugin and dependency to work with MySQL on local server

### LINKED ISSUE #9: Get app to run on local database

### PR DESCRIPTION
**Pull request by shaargtz**
#What does this PR do?
- It sets up pom.xml to run things on a local server and also to establish the connection to MySQL

**Related Issues**
- Closes #9 

---
**Merge Checklist**
- [ ] Branch is up to date
- [ ] No code style issues

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -35,11 +35,17 @@
...
+      <groupId>mysql</groupId>
+      <artifactId>mysql-connector-java</artifactId>
+      <version>8.0.19</version>
+    </dependency>
   </dependencies>
 
   <build>
     <plugins>
-      <!-- Provides `mvn package appengine:run` for local testing
+      <!-- Provides `mvn package appengine:run` for testing on the cloud
```
Comment (OUMontiel) [inline pom.xml:48]: You changed the comment but not the plugin? Is the original comment wrong or why the change?

Comment (shaargtz) [reply pom.xml:48]: yeah I didn't change the plugin, it's just to differentiate that one is the dev server on appengine and the other is the local server on our laptops
